weltyt.py

```diff
@@ -4,3 +4,23 @@
 print("Welcome to tyton")
 conn = sqlite3.connect("TYTDB.db")
 
+cur = conn.cursor()
+
+def my_function(base):
```

Comment:
Please, name the function after what it does.
Pass cursor to it, open the file inside it.